app/components/search/search.controller.js

```diff
@@ -0,0 +1,27 @@
+(function () {
+  let SearchController = function (VideosService, StorageService, $mdToast) {
+
+    let ctrl = this;
+    ctrl.query = {
+      text: "GCp2vSNdWBA",
+      engine: "youtube"
+    };
+
+    this.search = (query) => VideosService.searchByQuery(ctrl.query).then((res) => ctrl.searchResults = res.data.items);
+
+    this.addVideo = (video) => {
+      StorageService.addVideo(video);
```

Comment:
chyba od tej akcji addVideo powinienes uzalezniac $mdToast.show()? a nie robic to niezaleznie od siebie?